Tests/BraveWalletTests/SelectAccountTokenStoreTests.swift

```diff
@@ -6,10 +6,18 @@
 import Combine
 import XCTest
 import BraveCore
+import Preferences
 @testable import BraveWallet
 
 @MainActor class SelectAccountTokenStoreTests: XCTestCase {
   
+  override func setUp() {
+    Preferences.Wallet.showTestNetworks.reset()
+  }
+  override func tearDown() {
+    Preferences.Wallet.showTestNetworks.reset()
+  }
+  
```

Comment:
For all of these test classes, I think it's better to just enable this flag in `setUp()` (so all tests in each `XCTestCase` run with test networks enabled) and call `reset()` in `tearDown()`; then we can remove the assignments in `setupStore()`.